int/api/html/front/website.html

```diff
@@ -180,6 +188,7 @@ <h3 class="stepper-title"></h3>
                         <thead>
                             <tr>
                                 <th scope="col">Website name</th>
+                                <th scope="col">Website Description</th>
```

Comment:
Why "Description" and not "description" ? 
Either we use uppercase everywhere, or lowercase everywhere:
```suggestion
                                <th scope="col">Website Name</th>
                                <th scope="col">Website Description</th>
```

```suggestion
                                <th scope="col">Website name</th>
                                <th scope="col">Website description</th>
```